src/plugins/intel_gna/transformations/transpose_to_pwl.hpp

```diff
@@ -0,0 +1,333 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <cmath>
+#include <stdexcept>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include "ngraph/pattern/matcher.hpp"
+#include <ngraph/opsets/opset8.hpp>
+#include <legacy/ngraph_ops/power.hpp>
+
+namespace GNAPluginNS {
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief TransposeToPwl transformation replaces suitable activation function with pwl
```

Comment:
It'd be good to add information about what activation types are converted to PWL and what are not and why.

Reply:
We have to add a full description for the new transformation